base_station/gps/src/__main__.py

```diff
@@ -0,0 +1,51 @@
+import serial
```

Comment:
Can you add a docstring to the top of the file and for each function explaining what is going on from a high level or what is the purpose of the file/class/function?

Reply:
lmk if I have addressed this sufficiently in my latest commit

Reply:
I would still push for commenting the classes and functions but I will also accept it as it is now.

Reply:
What level of granularity are you looking for in the documentation? I am open to writing more especially if any part of my code is unclear